ios/Runner/Info.plist

```diff
@@ -2,6 +2,19 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+    <key>GIDClientID</key>
+    <string>615740456848-mv0jtsg5v88b56l0a89jlk94v5tlpemr.apps.googleusercontent.com</string>
+    <key>CFBundleURLTypes</key>
+    <array>
+    	<dict>
+    		<key>CFBundleTypeRole</key>
+    		<string>Editor</string>
+    		<key>CFBundleURLSchemes</key>
+    		<array>
+    			<string>com.googleusercontent.apps.615740456848-mv0jtsg5v88b56l0a89jlk94v5tlpemr</string>
+    		</array>
+    	</dict>
+    </array>
```

Comment:
Nie jestem fanem tego ze na publicznym repo wiszą sobie o tak klucze, nie wiem jak we flutterze to sie powinno robić ale w ustawieniach projektu na githubie można trzymać secrety.

Na czas developmentu niech to będzie ale potem trzeba będzie to przeresetować